### PR TITLE
Gv2 111 tags

### PR DESCRIPTION
**Pull request by AlinaKuzmenko**

### CODE REVIEW

File: src/components/Tag/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@stryberventures/gaia-react.tag",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "classnames": ">=2.0.0",
+    "react": ">=17.0.0",
+    "react-jss": ">=10.0.0",
+    "@stryberventures/gaia-react.theme": ">=0.0.1",
+    "@stryberventures/gaia-react.icons": ">=0.0.1"
```
Comment (SergeySychevskiy) [inline src/components/Tag/package.json:9]: Please, add Text as a dependency

Comment (AlinaKuzmenko) [reply src/components/Tag/package.json]: added

File: src/components/Tag/Tag.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  size: 'medium',
+  shape: 'square',
+  onRemove: undefined,
+};
+
+export const LeftIcon = Template.bind({});
+LeftIcon.args = {
+  children: 'Square removable',
+  iconLeft: <LeftArrow />,
+  onRemove: undefined,
```
Comment (SergeySychevskiy) [inline src/components/Tag/Tag.stories.tsx:63]: Since onRemove is optional, do you really need to pass it?

File: src/components/Tag/index.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  color?: 'primary' | 'secondary';
+  size?: TTagSize;
+  shape?: 'square' | 'round';
+  selected?: boolean;
+  disabled?: boolean;
+  iconLeft?: React.ReactNode;
+  iconRight?: React.ReactNode;
+  testId?: string;
+  className?: string;
+  onSelect?: (data?: any) => any;
```
Comment (SergeySychevskiy) [inline src/components/Tag/index.tsx:20]: Please provide the type of parameter and return value for both functions

File: src/components/Tag/Tag.stories.tsx (hunk 2 above)
Comment (AlinaKuzmenko) [reply src/components/Tag/Tag.stories.tsx:63]: It's for Storybook. If I don't do this, onRemove is never undefined within the Tag component.

Screenshot 2022-12-21 at 17 20 41

File: src/components/Tag/index.tsx (hunk 3 above)
Comment (AlinaKuzmenko) [reply src/components/Tag/index.tsx]: done

**Review by SergeySychevskiy — DISMISSED** [review]

File: src/components/Tag/index.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    selected = false,
+    children,
+    testId,
+    className,
+    onSelect,
+    onRemove,
+    ...rest
+  } = props;
+  const classes = useStyles(props);
+  const handleOnRemove = (event: any) => {
```
Comment (SergeySychevskiy) [inline src/components/Tag/index.tsx:40]: Please change the type of event

File: src/hooks/useKeyPress.ts [hunk 5]
```diff
@@ -2,9 +2,10 @@ import { useEffect } from 'react';
 
 export enum KEYS {
   esc = 'Escape',
+  enter = 'Enter',
 }
 
-export function useKeyPress (targetKey: string, callback: () => void) {
+export function useKeyPress (targetKey: string, callback: (data?: any) => void) {
```
Comment (SergeySychevskiy) [inline src/hooks/useKeyPress.ts:8]: From what I see - callbask doesn't have parameters, so we can remove (data?: any)

**Review by SergeySychevskiy — APPROVED** [review]

**Pull request merged**